mffpy/epoch.py

```diff
@@ -35,6 +36,14 @@ def add_block(self, duration):
         self.lastBlock += 1
         self.endTime += duration
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name
+
```

Comment:
[required]  Sorry, that's not really what I meant.   I meant to have a property `name` associated with the class such as:

```python
class MyClass:
    name = "Epoch"
    def __init__(self..):
...
```

The benefit of this would be that we can customize the default name of "Epoch" easily by doing
```python
from xml_files import Epochs
Epochs.name = "Default Epoch"
```

Sorry again for being unclear

Reply:
> [required] Sorry, that's not really what I meant. I meant to have a property `name` associated with the class such as:
> 
> ```python
> class MyClass:
>     name = "Epoch"
>     def __init__(self..):
> ...
> ```

@jusjusjus I'm not quite following. Do you mean

```
class Epoch:
    name = 'epoch'
```

or

```
class Epochs:
    name = 'epoch'
```

If it's the first one, I am not seeing how this would allow us to change the default epoch name for an `Epochs` object.

Reply:
First one.

Reply:
Ok got it. That makes sense now.